Resources/Prototypes/Entities/Structures/Furniture/potted_plants.yml

```diff
@@ -265,3 +265,16 @@
   components:
   - type: Sprite
     state: plant-29
+
+- type: entity
+  id: DimensionPot
+  parent: PottedPlant26
+  suffix: Dimension
+  components:
+  - type: DimensionPot
+  # don't want the pot going into its own portal do we
+  - type: PortalBlacklist
```

Comment:
Add this to construction objects so you can't drag things in like wall girders

Reply:
you can still build anything inside so what would this intend to prevent

Reply:
would preventing anchoring things inside instead be fine? you would be able to drag things like vendomats inside anyway so i think that would work even better as a solution

Reply:
Works for me